classification/models/classification_groups.py

```diff
@@ -371,19 +371,25 @@ def __lt__(self, other):
     def conditions(self) -> List[ConditionResolved]:
```

Comment:
(Note this is a left over from my code that made things more complicated than they need to be)

It might be cleanest to change the signature of ClassificationGroup.conditions to return a single ConditionResolved, as it only returns multiple now because a single ConditionResolved doesn't properly handle the merging of resolved terms and plain text - but they wont be grouped together anyway.

That way we don't need to hack the join term so much either (as in we can just use the condition tag as it was)